smoke-tests/spec/concourse_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "spec_helper"
+
+describe "concourse-test", "eks-manager": true do
+  context "concourse namespace should exists" do
+    it "Fail if namespace doesnt exist" do
+      expect(namespace_exists?("concourse")).to eq(true)
+    end
+  end
+
+  context "concourse-main namespace should exists" do
+    it "Fail if namespace doesnt exist" do
```

Comment:
I'd probably remove the "contexts" here.

You usually add a context when you need to have different setups for different groups of tests. But, in this case, you're just testing for the existence of some namespaces, so the contexts don't add value.

Also, I'd probably change this
```
    it "Fail if namespace doesnt exist" do
      expect(namespace_exists?("concourse")).to eq(true)
```
to this:
```
  specify {
      expect(namespace_exists?("concourse")).to eq(true)
  }
```

`specify` and `it` are synonyms in rspec, and the code itself is simple enough that the descriptive text doesn't really make it any easier to read.

Reply:
done